src/tactic/ui/panel/simple_side_bar_wdg.py

```diff
@@ -253,7 +253,23 @@ def get_link_wdg(my, element_name, config, options, info):
                 'link': link,
                 'target': target,
                 'cbjs_action': '''
-                var content = $(document).getElement(bvr.target);
+
```

Comment:
Yes, they are mootools class selectors.  Refer to mootools to see what you can do, however, I have never really used # (id selector).